docs/application/tizen-studio/release-notes/5-0-release-notes.md

```diff
@@ -0,0 +1,74 @@
+# Tizen Studio 5.0 Release Notes
+
+- Release Date: Oct 31, 2022
+
+## IDE and tools
+
+### New feature
+
+- WHPX support on Windows [Experimental]
+  - WHPX support has been added in Tizen emulators. Users can now use either HAXM or WHPX accelerators to launch Tizen emulators on Windows.
+- macOS notarization support for Tizen Installer
+  - Tizen installers are now officially notarized with Apple. Users can install Tizen Studio on macOS without any notarization errors.
+- Ubuntu 22.04 support
+  - Tizen Studio is now supported on Ubuntu 22.04.
+  - Visual Studio Code Extension for Tizen is now supported on Ubuntu 22.04.
+- CLI
+  - Support for non-Samsung vendor TV devices.
+  - Added support for RPK projects.
+- VSCode
+  - Added support for RPK projects.
+- Visual Studio (Windows)
+  - Added support to install Tizen dotnet workload in non-admin mode.
+  - Intellisense support has been added for Tizen web project config.xml file and workspace YAML file.
+
+### Fixed bugs
+  - Added the option in CLI to include the files outside app directory while packaging.
+  - Fixed the error of on-demand packages not getting installed in Visual Studio when multiple target devices are launched.
+  - Fixed the issue of multi app projects not getting packaged in CLI if static lib is the main project. 
+  - Fixed the issue of directories not being created in CLI packaging command when -o option is passed.
+  - Valgrind profiling support has been removed from Tizen Studio as its not supported in Tizen platform anymore.
+  - IoT Setup Manager support has been removed from Tizen Studio.
+
+### Known issues
+
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.

```

Comment:
```suggestion
  - On macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.

```